### ISSUE TITLE
Final specification for the accessibility terms?

### ISSUE THREAD

**Issue by iherman** [w3c/wpub#300]
I am trying to pick up the separate thread from #293 how these should be defined in our draft. Is it so that:

* `accessibilitySummary` is a localizable text
	* the current specification in the draft and the WebIDL are fine
* all the other terms, ***except*** for `accessModeSufficient`, are a (possible) _array_ of strings, each strings are allowed to take a value specified for that term. That means that the
	* the draft should be changed in the table specifying 'one or more texts'...
	* the current WebIDL (`sequence<DOMString>`) is fine
* `accessModeSufficient` is:
	* one or more texts, each a comma-separated list of values, each value being one of the predefined values
	* the WebIDL should be: `sequence<sequence<DOMString>>`

Knowing that this does not correspond to the schema.org documentation, but that is being taken care of in a separate discussion outside of this WG.

Is this a correct interpretation?

@mattgarrish @avneeshsingh @HadrienGardeur

**Comment by iherman** [comment]
I have prepared a version with these definitions: https://github.com/w3c/wpub/tree/accessibility-terms-update

@mattgarrish if that does reflect what is meant, you can merge this into the master; I will be off soon...

**Comment by HadrienGardeur** [comment]
I'll need to spend more time on `accessModeSufficient`, right now this isn't handled correctly yet in our JSON Schema.

**Comment by iherman** [comment]
Thanks @avneeshsingh 

@HadrienGardeur is it o.k. if we update the draft along these lines?
